tests-invoke

```diff
@@ -93,8 +93,15 @@ def main():
                 time.sleep(60)
         return_code = p.returncode
         sys.stderr.write("Test run finished, return code: {0}\n".format(return_code))
-        if not opts.pull_number:  # Create an issue
-            ...  # TODO implement
+        # Nightly tests use --pull-number 0
+        if opts.pull_number == "0" and return_code != 0:
```

Comment:
Any chance this could still be `if not opts.pull_number`, so that it could be `None`? That would be cleaner than `"0"`.